core/src/test/resources/com/google/errorprone/bugpatterns/NullablePrimitiveNegativeCases.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.errorprone.bugpatterns;
+
+import javax.annotation.Nullable;
+
+/**
+ * @author sebastian.h.monte@gmail.com (Sebastian Monte)
+ */
+public class NullablePrimitiveNegativeCases {
+
+  @Nullable
+  Integer a;
+
+  public void method(@Nullable Integer a) {
```

Comment:
Can you add positive and negative tests for methods annotated with @Nullable, to make sure the return type is tested correctly?